src/game/common/audio/audioeventrts.cpp

```diff
@@ -32,7 +32,6 @@ AudioEventRTS::AudioEventRTS() :
     m_priority(PRIORITY_NORMAL),
     m_volumeAdjustFactor(-1.0f),
     m_timeOfDay(TIME_OF_DAY_AFTERNOON),
-    m_objectID(),
```

Comment:
Why are `m_objectID` and `m_drawableId` not initialized here?

Reply:
Because they are members of a union and the compiler will handle the union initalization.